### ISSUE TITLE
The first time an EKS cluster is created its provider/distro is initially shown as imported/imported

### ISSUE THREAD

**Issue by mantis-toboggan-md** [rancher/dashboard#12883]
**Setup**
- Rancher version: 	v2.11-d224815d7cfbf4194db1ffdfb04676d55d1a2062-head
- Rancher UI Extensions: none
- Browser type & version: firefox

We may need to tackle https://github.com/rancher/dashboard/issues/12882 first.

**Describe the bug**
Originally reported [here](https://github.com/rancher/dashboard/issues/12662#issuecomment-2537465795). The first time a user creates an EKS cluster, its provider/distro in the cluster management cluster list view is shown as imported/imported.

**To Reproduce**
1. Bring up a new rancher instance
2. create an EKS cluster. Default values are fine.
3. Observe the cluster list view immediately after creating the cluster

**Result**
The new cluster's provider/distro is imported/imported

**Expected Result**
The new cluster's provider/distro should be `Amazon EKS`

**Screenshots**
![image](https://github.com/user-attachments/assets/ebad8272-b94e-4c7a-868b-20477a994c72)

**Additional context**
It seems that the mgmt cluster `status.driver` and `status.provider` are undefined initially. I think this bug is only obvious on the first EKS cluster because that first cluster is slower to provision. If I open a second window to watch the table as I create an EKS cluster, I do see it listed as imported/imported,  but only for 1-2 seconds at most.

**Comment by nwmac** [comment]
What should we do @mantis-toboggan-md ? If they are undefined and the cluster is provisioning, should we show `-`?

**Comment by gaktive** [comment]
This looks like it'll have to move to 2.12.0 with a backport for 2.11.1

**Comment by mantis-toboggan-md** [comment]
>If they are undefined and the cluster is provisioning, should we show -?

Yes, I think this is the best way. We should still assume a cluster is imported if provider/distro are undefined when its active

**Comment by eva-vashkevich** [comment]
I think this is caused by this https://github.com/rancher/dashboard/blob/13632bea2213eab25d8439e9e4c67204f685950a/shell/models/management.cattle.io.cluster.js#L94 line. But I think we need to look into it more holistically first, because changing this might have a lot of side effects

**Comment by gaktive** [comment]
If this stays in 2.14.0, a 2.13.x back port would be required but since this is also in the Ice Box, this can wait until we pick it up.

**Comment by gaktive** [comment]
New backports would need to be filed.
